README.md

```diff
@@ -54,6 +56,7 @@ The available methods added to `ENV`:
 * `integer` - produces an integer from the environment variable, by calling `to_i` on it (if it's
   present). Note that this means that providing a value like "hello" means you'll get `0`, since
   that's what ruby does when you call `"hello".to_i`.
+* `file_handle` - produces a `Pathname` object if the path name given by the environment variable matches an existing path.
```

Comment:
(It looks like it's actually called `file_path` in the code)

Reply:
it is 🤦🏼